src/js/containers/dashboard/DashboardSummaryContainer.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * DashboardSummaryContainer.jsx
+ * Created by Daniel Boos 11/13/19
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import * as DashboardHelper from 'helpers/dashboardHelper';
+import DashboardSummaries from 'components/dashboard/summary/DashboardSummaries';
+
+const propTypes = {
+    appliedFilters: PropTypes.object.isRequired
+};
+
+export class DashboardSummaryContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            results: [],
+            inFlight: false
+        };
+
+        this.getSummary = this.getSummary.bind(this);
+    }
+
+    componentDidMount() {
+        this.getSummary();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps !== this.props) {
```

Comment:
Could you use [lodash isEqual](https://lodash.com/docs/4.17.15#isEqual) here to make sure we're doing a deep comparison of the nested objects?